lib/meilisearch/index.rb

```diff
@@ -514,5 +514,20 @@ def update_non_separator_tokens(non_separator_tokens_attributes)
     def reset_non_separator_tokens
       http_delete("/indexes/#{@uid}/settings/non-separator-tokens")
     end
+
+    ### SETTINGS - PROXIMITY PRECISION
+
+    def proximity_precision
+      http_get("/indexes/#{@uid}/settings/proximity-precision")
+    end
+    alias get_proximity_precision proximity_precision
```

Comment:
I believe these `get_` aliases should be removed. They were requested to be removed in two of my previous PRs: https://github.com/meilisearch/meilisearch-ruby/pull/493#discussion_r1348725988

Reply:
Changed